app/Helpers/PrecinctImporter.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace App\Helpers;
+use Akeneo\Component\SpreadsheetParser\SpreadsheetParser;
+use App\City;
+use App\Precinct;
+use Illuminate\Support\Facades\File;
+
+class PrecinctImporter
+{
+
+    public function importFromFile(\Symfony\Component\HttpFoundation\File\File $file, bool $deleteFileAfter)
+    {
+        $fileExtension = $file->getExtension();
+        $isCsv = $fileExtension == "csv";
+        $isXlsx = $fileExtension == "xls" || $fileExtension == "xlsx";
+        $data = array();
+
+        if ($isCsv){
+            $data = $this->readFromCSV($file);
+        } else if ($isXlsx){
+            $data = $this->readFromXLSX($file);
+        }
+
+        $this->importPrecinctsFromArray($data);
+        if ($deleteFileAfter) {
+            File::delete($file);
+        }
+    }
+
+    private function readFromCSV(\Symfony\Component\HttpFoundation\File\File $file)
```

Comment:
import the class `\Symfony\Component\HttpFoundation\File\File` and use the simplified FQN (`File`) (for keeping the usages of type hints as in rest of the project)